repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -72,12 +99,10 @@ def _create_detected_instances_list(details):
 
 
 def _min_ver_string():
-    if version.get_target_major_version() == '8':
+    if version.matches_target_version('8.6'):
```

Comment:
This will return SAP_HANA_RHEL90_MINIMAL_VERSION_STRING when called for >=8.7 and <8.6  target versions. Don't think you want that like this. This needs to be checked properly that this is not a problem. However reading the code out of context is confusing.

Reply:
no, because we can line 101 only if target_version <=8.6, see lines 200-203
plus <8.6 is not allowed for HANA case.